src/main/kotlin/org/rust/lang/core/types/infer/TypeInference.kt

```diff
@@ -534,27 +535,27 @@ private class RsFnInferenceContext(
 
     private fun tryCoerce(inferred: Ty, expected: Ty): Boolean {
         return when {
-        // Coerce array to slice
+            // Coerce array to slice
```

Comment:
Hmm, intellij auto-format thinks differently about this

Reply:
It's weird but I've reverted this change